### ISSUE TITLE
Feature Request: Drop animation

### ISSUE THREAD

**Issue by dan-lee** [formkit/drag-and-drop#25]
Would be nice to have this feature: When you release a drag and drop item, it animates to the final position instead of jumping immediately.

**Comment by sashamilenkovic** [comment]
@agotfredsen82 Absolutely. I am hoping to get a new release out this weekend. After that, I'll be turning my attention to animations in general.

**Comment by justin-schroeder** [comment]
Status update: drop/settle animations are still planned and this remains the tracking issue. The animations plugin got incremental love in 0.6.0 (configurable `easing`, #190), and the dropOrSwap-can't-animate question (#164) was folded in here, since a drop animation is the right answer for plugins that don't live-sort. It's next in line for the animations work after the 0.6.0 correctness release ships.
